packages/gatsby-transformer-javascript-static-exports/src/gatsby-node.js

```diff
@@ -15,18 +15,30 @@ async function onCreateNode({ node, getNode, actions, loadNodeContent }) {
   const options = {
     sourceType: `module`,
```

Comment:
I'll admit i'm fairly unfamiliar with what this option does

Reply:
Until the community starts properly transitioning to using `.mjs` files, Babel needs to either told what type of files it is going to get, or it needs to be told to guess. Parsing as a `module` ends up meaning it is auto-strict, among other things. Setting `unambiguous` basically means it will attempt to autodetect the type, but might get in wrong in some cases. It should avoid the need for `strictMode: false` though since it'll only be auto-strict if the user used import or export statements.